open-sauced/intro
# feature: add screenshot of completed pr template to the how to contribute to o…

…pen source md file

## Description

This PR adds a screenshot of a completed pr template to the how to contribute to open source md file. 


## Related Tickets & Documents

Fixes #244


## Mobile & Desktop Screenshots/Recordings

![screenshot 1](https://github.com/user-attachments/assets/3bf3471e-c686-4ce2-b718-d3348fc50528)

## Steps to QA

1. Run the application
2. Click the Start the Intro to Open Source Course button
3. In the menu on the left click on the How to contribute to Open Source link
4. Scroll down to the Let's Get Practical section
5. Scroll down to 11 and you will see the Complete Pull Request Template Screenshot after the warning.


## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?

![image](https://github.com/user-attachments/assets/be155952-c9d4-4a6f-8dce-9729dbf0fdbc)
